projects/plugins/jetpack/modules/masterbar/admin-menu/class-wpcom-admin-menu.php

```diff
@@ -14,14 +14,24 @@
  */
 class WPcom_Admin_Menu extends Admin_Menu {
 
+	/**
+	 * WPcom_Admin_Menu constructor.
+	 */
+	protected function __construct() {
+		parent::__construct();
+
+		$this->customize_slug = 'https://wordpress.com/customize/' . $this->domain;
+	}
```

Comment:
Can the property be overriwritten out of the constructor? That would simplify the code, but TBH I don't know if this is possible in PHP:

```suggestion
	protected $customize_slug = 'https://wordpress.com/customize/' . $this->domain;
```

Reply:
Scratch that. I think it's not possible, because we cannot set a default variable which depends on another var like `$this->domain`